app/views/billing_expenses/show.html.slim

```diff
@@ -1,29 +1,84 @@
 .row
-  .col-xs-12
-    .table-responsive
-      table.table.table-no-border-top
-        tbody
-          tr
-            td= t_model(Volunteer)
-            td= link_to @volunteer.contact.full_name, volunteer_path(@volunteer)
-          tr
-            td= t_attr(:address)
-            td= @volunteer.contact.full_address
-          tr
-            td= t_attr(:bank_details, Volunteer)
-            td= @billing_expense.full_bank_details
-          tr
-            td= t_attr(:waive, Volunteer)
-            td= t(@volunteer.waive)
-          tr
-            td= t_attr(:amount)
-            td= @billing_expense.amount
-          tr
-            td= t_attr(:created_at)
-            td= l(@billing_expense.created_at.to_date)
-          tr
-            td= t_attr(:created_by)
-            td= @billing_expense.user.profile.contact.full_name
+  br=
+  br=
+  .col-xs-11.col-xs-offset-1
+    h3
+      strong
+        = "#{t('billing_expense_pdf.title')} "
+        = "#{@billing_expense.volunteer.contact.full_name} "
+        - if @volunteer.hours.any?
```

Comment:
@chrysalag should I leave this line anyway? for ensuring it doesn't crash? 
or is in the new automatic calculation ensured/required to have entered hours before creating it?

Reply:
Yes, but both this and the next line should be connected with this billing expense. Make a method to check if there are hours with billing_expense this billing expense and then take the last for the next line.